commons/kafka/record_set.avsc

```diff
@@ -0,0 +1,14 @@
+{
+    "namespace": "org.radarcns.kafka",
+    "name": "RecordSet",
+    "type": "record",
+    "doc": "Abbreviated record set, meant for binary data transfers of larger sets of data. It can contain just a source ID and the record values. The record keys are deduced from authentication parameters. This method of data transfer requires that the data actually adheres to the schemas identified by the schema version.",
+    "fields": [
+        {"name": "keySchemaVersion", "type": "int", "doc": "Key schema version for the given topic."},
+        {"name": "valueSchemaVersion", "type": "int", "doc": "Value schema version for the given topic."},
+        {"name": "projectId", "type": ["null", "string"], "doc": "Project ID of the sent data. If null, it is attempted to be deduced from the credentials.", "default": null},
+        {"name": "userId", "type": ["null", "string"], "doc": "Source ID of the sent data. If null, it is attempted to be deduced from the credentials.", "default": null},
```

Comment:
Typo in docs. Should be Subject Id or User Id. 
It would also be good to include the URL of gateway source file where this is used for more info in the record level doc